Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -88,6 +88,7 @@ namespace Babylon
             bgfx::setViewClear(m_viewId, m_flags, Color(), m_depth, m_stencil);
             // discard any previous set state
             bgfx::discard();
+            bgfx::touch(0);
```

Comment:
same here. is there a reasoning for this?

Reply:
That's the fix! I've updated this issue https://github.com/BabylonJS/BabylonNative/issues/118

Reply:
Sure, but why is it necessary?

Reply:
Removed.